CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+:tada:Thanks for taking the time to contribute!:tada:
+
+The following is a set of guidelines for contributing to hiredis-cluster.
+
+## Coding conventions
+
+### Code style
+
+Adhere to the existing coding style and make sure to mimic best possible.
+
+### Code formatting
+
+To have a common look-and-feel [clang-format](https://clang.llvm.org/docs/ClangFormat.html)
+is used for code formatting. The formatting rules can be applied to the
+source code by running following make target:
+
+```sh
+$ make format
+```
```

Comment:
Do you run this in the build directory? Maybe it should be mentioned that you need to run cmake first then?

How about mentioning that you should run `make test` as well, or is that obvious from the README?

Since the CI build complains about formatting and failing tests, they don't really have to run anything in advance, but it's good to know how to do it.

Reply:
Added a mention about build directory, and for now; a reference to the README about build/test